assets/scss/lib/partials/_homepage.scss

```diff
@@ -103,6 +133,34 @@
 
     ul {
       li {
+        display: block;
+        width: auto;
+        padding: 0 0 0 1rem;
+        position: relative;
+
+        @include rem(margin-bottom, 8);
+
+        &:not(:last-child):after { // remove list commas
+          content: '';
+        }
+
+        &:before {
+          content: $fa-chevron-right;
+          position: absolute;
+          left: 0;
+          top: 0.125rem;
+          font-family: $font-family-icons;
+          font-weight: $fa-font-weight;
+          color: $tertiary;
+          font-size: $font-size-ex-ex-sm;
```

Comment:
That's an interesting variable name :)

Reply:
yes we have a couple of these - there's also an ex-ex-large